### PR TITLE
Added Twitter Follow Badge

### PR DESCRIPTION
**Pull request by vinitshahdeo**
Prettified `README.md` with this badge:

Twitter Follow

### CODE REVIEW

**Comment by vinitshahdeo** [comment]
@RDIL Moved the badges to the top.

File: README.md [hunk 1]
```diff
@@ -108,4 +110,7 @@ Please note that Open Source Design has a [Contributor Code of Conduct](https://
 
 ## 📜 License
 
-- **🔀 You can use & modify everything as long as you credit [Open Source Design](https://opensourcedesign.net) and use the same license for your resulting work.** [Code license is AGPLv3](https://www.gnu.org/licenses/agpl-3.0.en.html) and content is [Creative Commons Attribution-ShareAlike](https://creativecommons.org/licenses/by-sa/4.0/).
+**🔀 You can use & modify everything as long as you credit [Open Source Design](https://opensourcedesign.net) and use the same license for your resulting work.** [Code license is AGPLv3](https://www.gnu.org/licenses/agpl-3.0.en.html) and content is [Creative Commons Attribution-ShareAlike](https://creativecommons.org/licenses/by-sa/4.0/).
+
+![Twitter Follow](https://img.shields.io/twitter/follow/opensrcdesign?style=social)
```
Comment (jancborchardt) [inline README.md]: The Twitter follow button should go with the other labels?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
+
 Website of the Open Source Design community, hosted on Github pages.
 
-[![Backers on Open Collective](https://opencollective.com/opensourcedesign/backers/badge.svg)](#backers) [![Sponsors on Open Collective](https://opencollective.com/opensourcedesign/sponsors/badge.svg)](#sponsors)
+[![Backers on Open Collective](https://opencollective.com/opensourcedesign/backers/badge.svg)](#backers) [![Sponsors on Open Collective](https://opencollective.com/opensourcedesign/sponsors/badge.svg)](#sponsors)
```
Comment (jancborchardt) [inline README.md]: This space seems unnecessary? :)

File: README.md [hunk 3]
```diff
@@ -1,8 +1,10 @@
 # [opensourcedesign.net](https://opensourcedesign.net)
 
+[![GitHub issues](https://img.shields.io/github/issues/opensourcedesign/opensourcedesign.github.io?logo=github)](https://github.com/opensourcedesign/opensourcedesign.github.io/issues) [![GitHub forks](https://img.shields.io/github/forks/opensourcedesign/opensourcedesign.github.io?logo=github)](https://github.com/opensourcedesign/opensourcedesign.github.io/network) [![GitHub stars](https://img.shields.io/github/stars/opensourcedesign/opensourcedesign.github.io?logo=github)](https://github.com/opensourcedesign/opensourcedesign.github.io/stargazers) [![GitHub license](https://img.shields.io/github/license/opensourcedesign/opensourcedesign.github.io?logo=apache)](https://github.com/opensourcedesign/opensourcedesign.github.io/blob/master/LICENSE)
```
Comment (jancborchardt) [inline README.md]: I feel like all of this info duplicates the GitHub interface? What’s the reasoning of adding it to the readme, when all of this info shows already at the top of the repository? :)

**Comment by RDIL** [comment]
Yeah, it seems more logical to not have these GitHub badges.  The license and twitter badges are fine though.

File: README.md [hunk 4]
```diff
@@ -1,8 +1,10 @@
 # [opensourcedesign.net](https://opensourcedesign.net)
 
+(https://github.com/opensourcedesign/opensourcedesign.github.io/stargazers) [![GitHub license](https://img.shields.io/github/license/opensourcedesign/opensourcedesign.github.io?logo=apache)](https://github.com/opensourcedesign/opensourcedesign.github.io/blob/master/LICENSE) ![Twitter Follow](https://img.shields.io/twitter/follow/opensrcdesign?style=social)
+
 Website of the Open Source Design community, hosted on Github pages.
 
-[![Backers on Open Collective](https://opencollective.com/opensourcedesign/backers/badge.svg)](#backers) [![Sponsors on Open Collective](https://opencollective.com/opensourcedesign/sponsors/badge.svg)](#sponsors)
+[![Backers on Open Collective](https://opencollective.com/opensourcedesign/backers/badge.svg)](#backers) [![Sponsors on Open Collective](https://opencollective.com/opensourcedesign/sponsors/badge.svg)](#sponsors)
```
Comment (RDIL) [inline README.md]:
```suggestion
[![Backers on Open Collective](https://opencollective.com/opensourcedesign/backers/badge.svg)](#backers) [![Sponsors on Open Collective](https://opencollective.com/opensourcedesign/sponsors/badge.svg)](#sponsors)
```

File: README.md [hunk 5]
```diff
@@ -1,5 +1,7 @@
 # [opensourcedesign.net](https://opensourcedesign.net)
 
+(https://github.com/opensourcedesign/opensourcedesign.github.io/stargazers) [![GitHub license](https://img.shields.io/github/license/opensourcedesign/opensourcedesign.github.io?logo=apache)](https://github.com/opensourcedesign/opensourcedesign.github.io/blob/master/LICENSE) ![Twitter Follow](https://img.shields.io/twitter/follow/opensrcdesign?style=social)
+
 Website of the Open Source Design community, hosted on Github pages.
 
 [![Backers on Open Collective](https://opencollective.com/opensourcedesign/backers/badge.svg)](#backers) [![Sponsors on Open Collective](https://opencollective.com/opensourcedesign/sponsors/badge.svg)](#sponsors)
```
Comment (jancborchardt) [inline README.md]: The Twitter follow badge should go after the Open Collective badges here. :) That way the description above is also not obscured by badges.

And again – the stargazers and license info we do not need as a badge, as it is shown directly on Github in the already very busy top area ;)
image

@vinitshahdeo could you make the changes and also squash your commits into 1? It’s 5 commits by now for just adding a Twitter button, we should keep history clean if possible. :)

**Comment by RDIL** [comment]
@jancborchardt I can fix and squash-merge it.

**Pull request merged**